storage/persistent_storage/persistent-storage-aws.adoc

```diff
@@ -6,7 +6,7 @@ include::modules/common-attributes.adoc[]
 toc::[]
 
 {product-title} supports AWS Elastic Block Store volumes (EBS). You can
-provision your {product-title} cluster with persistent storage using AWS EC2.
+provision your {product-title} cluster with persistent storage using link:https://docs.aws.amazon.com/AWSEC2/latest/UserGuide/concepts.html[Amazon EC2].
```

Comment:
Per ISG:

> using n
To avoid ambiguity, replace this gerund with either “by using” or “that uses.”